### DISCUSSION TITLE
higress脱离k8s的安装脚本依赖资源是否可以搞成国内可以快速访问的，比如镜像啥的，原因您懂的。。

### DISCUSSION THREAD

**Discussion by Everything1sPossible** [higress-group/higress#454] [General]
higress脱离k8s的安装脚本依赖资源是否可以搞成国内可以快速访问的，比如镜像啥的，原因您懂的。。

**Comment by Everything1sPossible** [comment]
安装命令一：使用独立部署的 Nacos目前没啥问题，安装命令二：使用 Higress 内置 Nacos拉取nacos镜像失败。

**Reply by CH3CHO** [reply]
nacos镜像拉取失败是什么错误呢？

**Reply by Everything1sPossible** [reply]
网络问题。。不过我换外部nacos了，安装命令一没啥问题

**Comment by johnlanni** [comment]
嗯 目前是放在 higress.io 上，您可以改成 higress.cn，这个就是国内镜像

**Reply by Everything1sPossible** [reply]
收到，谢谢
